README.md

```diff
@@ -1,3 +1,6 @@
+# RapidPliant
+This is a fork of the Pliant repository - with the purpose of refactoring and any other changes that I would personally like to have while working on other projects - this way making it easier to pull changes back to Pliant if anyone would want to.
+
```

Comment:
Nope, you can remove that - that wasn't really supposed to be in there with the pull request :) - newbie mistake :)